models/cxr_image_synthesis_latent_diffusion_model/configs/inference.json

```diff
@@ -0,0 +1,110 @@
+{
+    "imports": [
```

Comment:
Hi @virginiafdez , thanks for the PR. Could you also help to add a test file? The purpose is to ensure the bundle is runnable, and here is an example:
https://github.com/Project-MONAI/model-zoo/blob/dev/ci/unit_tests/test_spleen_ct_segmentation.py#L37